src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1383,13 +1383,21 @@ void MulticopterPositionControl::control_auto(float dt)
 	if (_pos_sp_triplet.current.valid) {
 
 		//only project setpoints if they are finite, else use current position
+		static bool first_nonfinite = true;
```

Comment:
Ok - can you change the static variable to a class member please? It makes the code very hard to read otherwise. Thanks!